packages/flutter/lib/src/material/slider.dart

```diff
@@ -434,7 +434,12 @@ class Slider extends StatefulWidget {
   /// {@macro flutter.widgets.Focus.autofocus}
   final bool autofocus;
 
-  final _SliderType _sliderType ;
+  /// The distance pointer has to travel in global coordinate system after
+  /// drag was confirmed to be active, for framework to be confident that user wants to drag the slider.
+  @visibleForTesting
+  static const double activeDragSlop = 6.0;
```

Comment:
Why 6?

Reply:
18.0 causes https://github.com/flutter/flutter/issues/40310
But 0.0 would not allow vertical drags at all

So I picked 6.0 as a compromise value to remove the "jump", but still let the vertical drag

Reply:
So, it still doesn't really fix https://github.com/flutter/flutter/issues/40310 then if you want to move even smaller distances.